Library/Usage.cs

```diff
@@ -189,13 +189,13 @@ internal void WriteXml(XmlTextWriter xmlWriter, bool update)
             xmlWriter.WriteElementString("merchant_tag", MerchantTag);
             
             if (RecordingTimestamp.HasValue)
-                xmlWriter.WriteElementString("recording_timestamp", RecordingTimestamp.Value.ToString("s"));
+                xmlWriter.WriteElementString("recording_timestamp", RecordingTimestamp.Value.ToUniversalTime().ToString("yyyy-MM-ddTHH:mm:ssZ"));
```

Comment:
Was there not a standard ISO8601 format code? why spell it out here?

Reply:
The [format code for "o"](https://docs.microsoft.com/en-us/dotnet/standard/base-types/standard-date-and-time-format-strings#the-round-trip-o-o-format-specifier) is the one called out in the Microsoft docs for this standard. If you use that, it looks like this `2018-10-10T07:30:20.0000000Z`

The only real difference is all of those trailing `0`s which I suppose isn't an issue. I wanted it to match the value coming back from Recurly exactly, but I can change the format to "o" if you think that makes more sense for this library.